trie-root/README.md

```diff
@@ -1,2 +1,2 @@
 This crate provides utility functions to validate and initialize tries using flexible input.
-It is used extensively in `parity-ethereum` to validate blocks (mostly transactions and receipt roots).
\ No newline at end of file
+It is used extensively in `substrate` to validate blocks (mostly transactions and receipt roots).
```

Comment:
is this statement correct or would you change it?

Reply:
It is correct.